plugins/mcreator-core/datalists/enchantmenttypes.yaml

```diff
@@ -1,14 +1,19 @@
-- BREAKABLE

```

Comment:
Keep the order in data list the same as it was

Reply:
Ok

Reply:
Done!